src/Transformer/DetermineHeroImages.php

```diff
@@ -31,25 +31,25 @@
 final class DetermineHeroImages implements Transformer {
 
 	/**
-	 * XPath query to find the existing hero images (elements with data/hero attribute).
+	 * XPath query to find the custom logo.
 	 *
 	 * @var string
 	 */
-	const EXISTING_HERO_IMAGES_XPATH_QUERY = './/*[@data-hero]';
+	const CUSTOM_HEADER_XPATH_QUERY = ".//*[ contains( concat( ' ', normalize-space( @class ), ' ' ), ' wp-custom-header ' ) ]//*[ ( self::img or self::amp-img ) and not( @data-hero ) ]";
```

Comment:
This query will match the header in Twenty Twelve, Twenty Fourteen, Twenty Sixteen and then any theme that uses `the_custom_header_markup()`:

```suggestion
	const CUSTOM_HEADER_XPATH_QUERY = ".//*[ @id = 'wp-custom-header' or @id = 'masthead' or @id = 'site-header' ]//*[ ( self::img or self::amp-img ) and not( @data-hero ) and not( contains( concat( ' ', normalize-space( @class ), ' ' ), ' custom-logo ' ) ) ]";
```

Note that it excludes any image which is the custom logo, which is needed for Twenty Sixteen and other themes that have a custom logo in the header area alongside the custom header.